docs/user/whats-new.asciidoc

```diff
@@ -92,5 +92,17 @@ You can also now
 create embedded visualizations in your *Canvas* workpads.
 Just like in *Dashboard*, you can promote favorites to the *Visualization Library* for re-use.
 
+[float]
+=== {ml-cap} model management
+
+You can now add more models to bring modern 
```

Comment:
I think we call them "trained models" most places, so perhaps we should do so the first time here too?:
```suggestion
You can now add more trained models to bring modern 
```